### PR TITLE
Add stride helpers and rework ibc

### PR DESCRIPTION
**Pull request by keyleu**
Added helpers for stride, reworked IBC (using the previous way I couldn't query a trace unless I did an IBC transaction first, but most times I need the IBC denom before doing  a transaction). Rest is linting and versioning

### CODE REVIEW

File: src/utils/setup/stride.rs [hunk 1]
```diff
@@ -0,0 +1,70 @@
+use localic_std::{errors::LocalError, transactions::ChainRequestBuilder};
+use log::info;
+use serde_json::Value;
+
+use crate::{utils::test_context::TestContext, ADMIN_KEY, STRIDE_CHAIN_NAME};
+
+pub fn set_up_host_zone(test_ctx: &mut TestContext, dest_chain: &str) {
```
Comment (lexzaiello) [inline src/utils/setup/stride.rs]: This should probably be a member function of the TestContext for consistency with the rest of the API. Same for other functions.

File: src/utils/setup/stride.rs [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+            &test_ctx.get_chain_prefix().src(dest_chain).get(),
+            &host_denom_on_stride,
+            &stride_to_host_channel_id,
+            ADMIN_KEY,
+        )
+        .unwrap();
+    }
+}
+
+pub fn query_host_zone(rb: &ChainRequestBuilder, chain_id: &str) -> bool {
```
Comment (lexzaiello) [inline src/utils/setup/stride.rs]: See previous comment about member functions.

**Review by lexzaiello — CHANGES_REQUESTED** [review]
Looks good! Thanks for the cleanup! Just one suggestion: the functions in stride should probably be member functions of the TestContext to keep consistency with the rest of the API. Besides that looks great!

File: src/utils/setup/stride.rs (hunk 1 above)
Comment (keyleu) [reply src/utils/setup/stride.rs]: Done!

File: src/utils/setup/stride.rs (hunk 2 above)
Comment (keyleu) [reply src/utils/setup/stride.rs]: Done!

**Review by lexzaiello — APPROVED** [review]
LGTM! 🙌

**Pull request merged**